internal/parser.go

```diff
@@ -2499,8 +2499,14 @@ func frontmatterIM(p *parser) bool {
 }
 
 func inExpressionIM(p *parser) bool {
-	// p.afe (active formatting elements) should be cleared when entering an expression
-	p.clearActiveFormattingElements()
```

Comment:
This was a regression introduced while trying to fix expressions with `<a>` tags!